src/main/webapp/app/iris/base-chatbot/iris-base-chatbot.component.ts

```diff
@@ -15,11 +15,16 @@ import { AccountService } from 'app/core/auth/account.service';
 import { animate, group, style, transition, trigger } from '@angular/animations';
 import { IrisChatService } from 'app/iris/iris-chat.service';
 import * as _ from 'lodash-es';
+import { ArtemisSharedComponentModule } from 'app/shared/components/shared-component.module';
+import { ChatStatusBarComponent } from 'app/iris/base-chatbot/chat-status-bar/chat-status-bar.component';
+import { ArtemisMarkdownModule } from 'app/shared/markdown.module';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
 
 @Component({
     selector: 'jhi-iris-base-chatbot',
     templateUrl: './iris-base-chatbot.component.html',
     styleUrls: ['./iris-base-chatbot.component.scss'],
+    standalone: true,
```

Comment:
```suggestion
    changeDetection: ChangeDetectionStrategy.OnPush,
    standalone: true,
```

Is there a reason that we avoid `OnPush` change detection strategy here?